inbm/dispatcher-agent/tests/unit/sota/test_sota.py

```diff
@@ -176,10 +176,13 @@ def _build_mock_repo(num_files=0):
                 mem_repo.add("test" + str(i + 1) + ".rpm", b"0123456789")
         return mem_repo
 
-    @patch("dispatcher.sota.downloader.Downloader.is_valid_release_date", return_value=False)
     @patch("inbm_lib.detect_os.detect_os")
-    def test_check_raise_exception(self, mock_detect_os, check_release_date):
+    def test_check_raise_exception(self, mock_detect_os):
         parsed_manifest = {'release_date': "1970-01-01"}
         mock_detect_os.return_value = 'Ubuntu'
         TestSota.sota_instance._parsed_manifest = parsed_manifest
-        self.assertRaises(SotaError, TestSota.sota_instance.check)
+        try:
```

Comment:
This test doesn't make any sense.  It says that it should raise, but we are failing if it raises.

Reply:
Since ubuntu check_release_date() method should return always True, we made the test case to fail if SOTA check() method raises exception.  Means in Ubuntu case check() method should not raise exception. 

Please correct me if my understanding is wrong.